can_tools/scrapers/official/MI/mi_vaccine_demographics.py

```diff
@@ -63,9 +63,19 @@ def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
 
         # combine detroit and wayne county into one location
         data["location_name"] = data["location_name"].replace("Detroit", "Wayne")
-        data = (
-            data.groupby(by=[col for col in data.columns if col != "value"])
-            .sum()
-            .reset_index()
-        )
-        return data
+
+        # sum over age and race columns to create independent demographic data
```

Comment:
I talked with Nick and he expressed that it would be helpful to have the demographic data in independent entries (instead of combinations of race and age) so this sums over the race and age columns to create individual demographic rows